src/presentation/pages/NoteSettings.vue

```diff
@@ -8,11 +8,20 @@
       :title="t('noteSettings.customHostname')"
       :placeholder="t('noteSettings.hostnamePlaceholder')"
     /> -->
-    <Checkbox
-      v-model:checked="noteSettings.isPublic"
-      :label="t('noteSettings.isPublic')"
-      @update:checked="changeAccess"
-    />
+    <Section
+      :title="t('noteSettings.availabilityTitle')"
+      :caption="t('noteSettings.availabilityCaption')"
+    >
+      <Row title="Publish Note">
```

Comment:
do we need a row here?

Reply:
nevermind, row is fine

Reply:
I think yes because it is a good template for this case: Row have title and right slot witch can be used for Checkbox component